Makefile

```diff
@@ -98,6 +105,7 @@ binary: build ## build the linux binaries
 	$(DOCKER_RUN_DOCKER) hack/make.sh binary
```

Comment:
@nishanttotla yes, it will do the same as usual, but will also have the bind-mounted binary specified by DOCKER_CLI_PATH. It will be added to your PATH so you can use it right away inside the dev container. Note that instead of DOCKER_CLI_PATH, you can also specify a different remote and git sha in hack/dockerfile/binaries-commits if you want to show case the full feature on one branch.